qa/run_xfstests.sh

```diff
@@ -46,7 +51,7 @@ TESTS="-g auto"	# The "auto" group is supposed to be "known good"
 
 # rbd presents geometry information that causes mkfs.xfs to
 # issue a warning.  This option avoids this class of problems.
-XFS_MKFS_OPTIONS="-l su=32k"
+#XFS_MKFS_OPTIONS="-l su=32k"
```

Comment:
ideally moving these options would be in a separate commit, but not a big deal. I'd merged it as is with the signed-off-bys added